firebase-firestore/src/androidMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -65,10 +56,28 @@ actual class FirebaseFirestore(val android: com.google.firebase.firestore.Fireba
 
     actual fun useEmulator(host: String, port: Int) {
         android.useEmulator(host, port)
-        android.firestoreSettings = FirebaseFirestoreSettings.Builder()
+        android.firestoreSettings = com.google.firebase.firestore.FirebaseFirestoreSettings.Builder()
             .setPersistenceEnabled(false)
             .build()
     }
+
+    actual fun setSettings(value: FirebaseFirestoreSettings) {
+        android.firestoreSettings = value.run {
+            com.google.firebase.firestore.FirebaseFirestoreSettings.Builder().also { builder ->
+                persistenceEnabled?.let { builder.setPersistenceEnabled(it) }
+                sslEnabled?.let { builder.isSslEnabled = it }
+                host?.let { builder.host = it }
+                cacheSizeBytes?.let { builder.cacheSizeBytes = it }
+            }.build()
+        }
+    }
+
+    actual suspend fun disableNetwork() {
+        android.disableNetwork().await().run {  }
+    }
+    actual suspend fun enableNetwork() {
+        android.enableNetwork().await().run {  }
```

Comment:
dont need .run { }

Reply:
I did this because I saw the same in clearPersistence() in this file

Reply:
only need if you use = instead of { ... }

Reply:
ah, gotcha!